### ISSUE TITLE
specify event logo size and dimensions

### ISSUE THREAD

**Issue by mattstratton** [devopsdays/devopsdays-web#217]
I don't know that we want to be explicit about them, but we should set a max width and then wrap the img resource with it.

This all gets better once we move to responsive/bootstrap.

**Comment by bridgetkromhout** [comment]
Okay. I think if we're recommending
```
 <div style="text-align:center;">
  {{< event_logo >}}
</div>
```

and giving no further guidance than that, then probably the default behavior needs to be non-giant size.

Example of badness:

![screen shot 2016-05-05 at 6 19 43 am](https://cloud.githubusercontent.com/assets/2104453/15042674/ae603356-128b-11e6-8750-1e313a6c4c11.png)

How's this band-aid?

**Comment by mattstratton** [comment]
resolved in #220
